es5.js

```diff
@@ -0,0 +1,74 @@
+function BuilderES5 (str) {
+    this.str = str;
+}
+
+function StringBuilder (str) {
+    BuilderES5.call(this, str);
+}
+    
+StringBuilder.prototype.plus = function (...str) {
+    this.str = [this.str, ...arguments].join('');
+    return this.str;
+}
+
+StringBuilder.prototype.minus = function (n) {
+    if (!this.str) {
+        throw "String is empty!";
```

Comment:
you should return a new Error object and pass your message inside of it. Please check `Error` object in js